### PR TITLE
some cleanup and set default memory to 4GB.

### PR DESCRIPTION
**Pull request by sashi-coursera**

### CODE REVIEW

File: coursera_autograder/commands/upload.py [hunk 1]
```diff
@@ -422,11 +419,11 @@ def setup_registration_parser(parser):
         '--grader-memory-limit',
         type=int,
         choices=list(range(2048, 16384, 1024)),
```
Comment (fng-coursera) [inline coursera_autograder/commands/upload.py:421]:
```suggestion
        choices=list(range(4096, 16384, 1024)),
```

File: coursera_autograder/commands/upload.py [hunk 2]
```diff
@@ -422,11 +419,11 @@ def setup_registration_parser(parser):
...
-        default = 2048,
+        default = 4096,
         help='Amount of memory your grader is allocated when grading '
              'submissions. Amount of memory you can request for 1 CPU is '
              'between 2048 MB to 8192 MB. For 2 CPUs is between 4096 MB to 16384 MB.'
```
Comment (fng-coursera) [inline coursera_autograder/commands/upload.py:425]: Do we also need to change the amount of memory available for 2 CPUs?
```suggestion
             'between 4096 MB to 8192 MB. For 2 CPUs is between 4096 MB to 16384 MB.'
```

File: README.rst [hunk 3]
```diff
@@ -109,7 +109,7 @@ timeout are all customizable.
    the grader will have access to when grading. The default is 1.
  - ``--grader-memory-limit`` takes a value between 2048 to 16384, increnment of 1024. representing the
```
Comment (fng-coursera) [inline README.rst:110]:
```suggestion
 - ``--grader-memory-limit`` takes a value between 4096 to 16384, increnment of 1024. representing the
```

**Comment by sashi-coursera** [comment]
Thanks @fng-coursera for review, and suggestions. I didn't know that feature. It is awesome. With few clicks, it is ready for re-review :).

**Review by fng-coursera — APPROVED** [review]

**Comment by fng-coursera** [comment]
> Thanks @fng-coursera for review, and suggestions. I didn't know that feature. It is awesome. With few clicks, it is ready for re-review :).

Yeah thanks to @ricky-wong for this 😄

**Pull request merged**